vid/src/advz.rs

```diff
@@ -63,7 +63,7 @@ pub type AdvzGPU<'srs, E, H> = AdvzInternal<
     E,
     H,
     (
-        HostOrDeviceSlice<'srs, IcicleAffine<<UnivariateKzgPCS<E> as GPUCommittable<E>>::IC>>,
+        DeviceVec<IcicleAffine<<UnivariateKzgPCS<E> as GPUCommittable<E>>::IC>>,
         CudaStream,
```

Comment:
do we strictly need `type AdvzGPU`? the `'srs` lifetime can be removed at least?

Reply:
Sure we could remove the lifetime specifier. I'm not sure what do you mean, `AdvzGPU` takes additional type generics for on GPU srs and CudaStream.

Reply:
> AdvzGPU takes additional type generics for on GPU srs and CudaStream.

i forgot about this. nvm. maybe just remove the lifetime.